<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryDataAnalyse</class>
 <widget class="QMainWindow" name="HistoryDataAnalyse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_date_end_selection">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_date_start_selection">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_query">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_observe_well">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>41</width>
      <height>9</height>
     </rect>
    </property>
    <property name="text">
     <string>观测井:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_start_date">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>41</width>
      <height>9</height>
     </rect>
    </property>
    <property name="text">
     <string>起始年份:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_month">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>41</width>
      <height>9</height>
     </rect>
    </property>
    <property name="text">
     <string>月份:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_month">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_end_date">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>41</width>
      <height>9</height>
     </rect>
    </property>
    <property name="text">
     <string>结束年份:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_observe_well_selection">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
